package/amazon-alexa/apl-core-library/0003-Add_install_methods.patch

```diff
@@ -1,21 +0,0 @@
-diff --git a/aplcore/CMakeLists.txt b/aplcore/CMakeLists.txt
```

Comment:
Does Yoga installs after removing this patch? It should be buildable by apl-core-library, but apl-client-library depends on Yoga in sysroot.

Reply:
Without this patch, apl-core does not install YOGA includes. But apl-client-libarary does not showing any compile issues without that. It is not using any Yoga related headers inside the code, but using yoga library. Do we need to add install support even though ?

Reply:
If it works, then it's ok